packages/client-core/src/i18n.ts

```diff
@@ -1,20 +1,15 @@
-import admin from '../i18n/en/admin.json'
-import common from '../i18n/en/common.json'
-import editor from '../i18n/en/editor.json'
-import social from '../i18n/en/social.json'
-import user from '../i18n/en/user.json'
+import { merge } from 'lodash'
```

Comment:
Do we need this? Can't we use Object.assign ?

Reply:
import { merge } from 'lodash'

	// @ts-ignore
	const resources = Object.entries(import.meta.globEager('../i18n/**/*.json')).reduce((obj, [key, translation]) => {
	  const [language, namespace] = key.replace('../i18n/', '').replace('.json', '').split('/')
	  const result = { [language]: { [namespace]: translation } }
	  return merge(result, obj)
	}, {} as any)

	export const getI18nConfigs = () => {
	  return {
		resources,
		namespace: Object.keys(resources['es'])
	  }
	}